pkg/handler/login.html

```diff
@@ -1,19 +1,60 @@
 <!doctype html>
-<html lang="en">
+ <html lang="en">
 
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Github Example</title>
-    <style>
-        a.button {
-            text-decoration: none;
-        }
-    </style>
-</head>
+ <head>
+     <meta charset="utf-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <title>Login with Github</title>
+     <style>
+         html, body {
+             background-color: #efefef;
+             display: flex;
+             flex-direction: column;
+             font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+             height: 100%;
+             justify-content: center;
+         }
 
-<body>
-    <a href="/github/login" class="button">Login with Github</a>
```

Comment:
Should it be left like this or changed?

Reply:
> Should it be left like this or changed?

Can be changed, just the url value needs to be `/github/login`.

Reply:
Done.